### PR TITLE
stf: Device name related updates

### PR DESCRIPTION
**Pull request by sirmordred**
hi 
if you have any objections or suggestions against PR, i can the update the code 

thanx

### CODE REVIEW

**Comment by sirmordred** [comment]
cc  @koral-- @sorccu @denis99999

File: lib/util/devutil.js [hunk 1]
```diff
@@ -159,12 +160,17 @@ devutil.makeIdentity = function(serial, properties) {
     model = model.substr(manufacturer.length)
   }
 
-  if (marketName) {
-    var devices = androidDeviceList.getDevicesByDeviceId(marketName)
-    if (devices.length > 0) {
-      marketName = devices[0].name
+  if (isNaN(customMarketName)) {
```
Comment (koral--) [inline lib/util/devutil.js:163]: NaN should be used only with numeric values.
`customMarketName` is rather a string so we can check whether it is defined. See eg. line 153.

**Comment by koral--** [comment]
Sign-off is needed in order for DCO check to pass.

File: lib/util/devutil.js (hunk 1 above)
Comment (sirmordred) [reply lib/util/devutil.js]: ahh good catch 👍  and done 👍

**Comment by sirmordred** [comment]
okay signed-off 👍

File: lib/util/devutil.js [hunk 2]
```diff
@@ -159,10 +160,15 @@ devutil.makeIdentity = function(serial, properties) {
...
 
-  if (marketName) {
-    var devices = androidDeviceList.getDevicesByDeviceId(marketName)
-    if (devices.length > 0) {
-      marketName = devices[0].name
+  if (customMarketName) {
+    marketName = customMarketName
+  }
+  else {
+    if (marketName) {
```
Comment (koral--) [inline lib/util/devutil.js:167]: Those 2 nesting levels can be merged to a single `else if`

Comment (sirmordred) [reply lib/util/devutil.js:167]: hmm okay thats right 👍  will do it

Comment (sirmordred) [reply lib/util/devutil.js]: okay done 👍

**Review by koral-- — APPROVED** [review]

**Comment by sirmordred** [comment]
@koral-- as we tried on our test phones, adb shell setprop failed to change ro.stf.product.name property, as i read it through here https://stackoverflow.com/questions/66877858/android-setprop-failed-to-set-property some prefixes are not allowed, so i changed it to "debug.stf.product.name" and it worked flawlessly 👍  credits to my colleague @yavuzyoldas

**Comment by sirmordred** [comment]
ready to go 👍

**Review by koral-- — APPROVED** [review]

**Pull request merged**